commit/merkleroot/types.go

```diff
@@ -5,20 +5,14 @@ import (
 
 	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 
+	"github.com/smartcontractkit/chainlink-ccip/commit/merkleroot/rmn"
 	"github.com/smartcontractkit/chainlink-ccip/plugintypes"
 	"github.com/smartcontractkit/chainlink-ccip/shared"
 )
 
 type Query struct {
-	RmnOnRampMaxSeqNums []plugintypes.SeqNumChain
-	MerkleRoots         []cciptypes.MerkleRootChain
-}
-
-func NewCommitQuery(rmnOnRampMaxSeqNums []plugintypes.SeqNumChain, merkleRoots []cciptypes.MerkleRootChain) Query {
-	return Query{
-		RmnOnRampMaxSeqNums: rmnOnRampMaxSeqNums,
-		MerkleRoots:         merkleRoots,
-	}
+	RetryRMNSignatures bool
+	RMNSignatures      *rmn.NodeSignatures
```

Comment:
Another advantage of using the protos - we can just plug in `ReportingPluginQuery` here directly, which has signatures and `FixedDestLaneUpdate` objects.

Reply:
Are you expecting these to change? Looks like the report tyeps changed on RMNRemote.sol in the latest batch of contract changes

![image](https://github.com/user-attachments/assets/3383ceec-ddde-4ce5-966d-406fcad25272)

Reply:
I don't really expect them to change